app/services/proofing/aamva/response/verification_response.rb

```diff
@@ -104,7 +114,8 @@ def parse_response
         end
 
         def rexml_document
-          @rexml_document ||= REXML::Document.new(http_response.body)
+          return @rexml_document if @rexml_document
+          @rexml_document = REXML::Document.new(http_response.body)
```

Comment:
could `REXML::Document.new` ever return `nil` or something falsy? (my guess is no, that is really rare for a constructor)

If it does not return nil, I think the `||=` syntax is better